cmd/identity/keys.go

```diff
@@ -4,22 +4,41 @@
 package main
 
 import (
+	"archive/tar"
```

Comment:
Any reason you aren't just writing them to a folder instead of tarring?

Reply:
Anyone using it, can later compact them as needed -- also that would make the code much simpler as well.

Reply:
It would definitely make the code simpler; honestly, I hadn't considered it. I think it's cleaner to have one file per identity though and makes it less likely that you make a mistake when transporting or using them.

Reply:
I think anyone using batch-generate should have sufficient knowledge to understand that the whole directory is important.